src/test/java/org/opentripplanner/updater/trip/TimetableSnapshotSourceTest.java

```diff
@@ -56,7 +57,7 @@ public class TimetableSnapshotSourceTest {
   private final GtfsRealtimeFuzzyTripMatcher TRIP_MATCHER_NOOP = null;
 
   private final boolean fullDataset = false;
-  private LocalDate serviceDate;
+  private final LocalDate SERVICE_DATE = LocalDate.parse("2009-02-01");
```

Comment:
it should probably be static and moved up in the code to follow code conventions

Reply:
Good point. Done in https://github.com/opentripplanner/OpenTripPlanner/pull/4797/commits/cbd2ff745d1ef21351c96ac587310a8cfa3835e1